framework/jaif-inference-scripts/infer-and-annotate.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# This script receives as arguments:
+# 1. Processor name.
+# 2. Classpath (target project's classpath and annotation-file-utilities.jar path)
+# 3. Project source folder path
+# 4. Extra processor arguments
+
+# Example of usage:
+#./infer-and-annotate.sh org.checkerframework.common.value.ValueChecker $PROJECT_CP:..../annotation-file-utilities.jar project_src_folder/ $EXTRA_PROCESSOR_ARGS
+# $PROJECT_CP must contain a path to annotation-file-utilities.jar. In case of 
+# using this script for Android projects, you must include paths to:
+# android.jar, gen folder, all libs used, source code folder.
+# The Android project must be built with "ant debug" before running this script.
+
+function run_cf_javac() {
+    processor=$1
+    cp=$2
+    source_files_dir=$3
+    extra_args=$4
+    cp=$cp:${CHECKERFRAMEWORK}/../annotation-tools/annotation-file-utilities/annotation-file-utilities.jar
+    files_to_compile=""
+    for f in $(find $source_files_dir/ -name "*.java"); do
+        files_to_compile="$files_to_compile $f"
```

Comment:
This will not work correctly if the filenames contain spaces.  (Mac and Windows users do that.)  Perhaps it's better to use find's -print0 argument.